src/routes/error/NotFoundView.vue

```diff
@@ -0,0 +1,42 @@
+<script setup>
```

Comment:
Super barebones, just need someone to pretty it up.

![image](https://github.com/cuttle-cards/cuttle/assets/414475/0ce3811a-2d75-4fef-9562-e3bf84d126f7)